great_expectations_cloud/agent/cli.py

```diff
@@ -36,26 +41,53 @@ def _parse_args() -> Arguments:
         default=False,
         type=pathlib.Path,
     )
+    parser.add_argument(
+        "--json-log",
+        help="Output logs in JSON format. Defaults to False.",
+        default=False,
+        action="store_true",
+    )
     parser.add_argument(
         "--log-cfg-file",
-        help="Path to a logging configuration json file. Supersedes --log-level and --skip-log-file.",
+        help="Path to a logging configuration file in JSON format. Supersedes --log-level and --skip-log-file.",
```

Comment:
Are we going to provide an example logging configuration file since this is a public repo?

Reply:
outside scope of this but I'll tt docs team